### PR TITLE
Fix url bugs in docs

### PR DESCRIPTION
**Pull request by saintmalik**
## Summary

Easy navigation and easy reading for others

### CODE REVIEW

**Comment by saintmalik** [comment]
@RaduBerinde can you look into this sir

File: v20.1/build-a-spring-app-with-cockroachdb-jpa.md [hunk 1]
```diff
@@ -622,7 +622,7 @@ Typically, Liquibase properties are defined in a separate [`liquibase.properties
 
 `Account.java` defines the [entity](https://en.wikipedia.org/wiki/Domain-driven_design#Building_blocks) for the `accounts` table. This class is used throughout the application to represent a row of data in the `accounts` table.
 
-Here are the contents of [`Account.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/Account.java):
+Here are the contents of [`Account.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/jpa/Account.java):
```
Comment (ericharmeling) [inline v20.1/build-a-spring-app-with-cockroachdb-jpa.md:625]: The url is now actually 

https://github.com/cockroachlabs/roach-data/tree/master/roach-data-jpa/src/main/java/io/roach/data/json/Account.java

File: v20.1/build-a-spring-app-with-cockroachdb-jpa.md [hunk 2]
```diff
@@ -642,7 +642,7 @@ Spring Data JPA supports standard Java Persistence API (JPA) annotations for dom
 
 To represent database objects as [HAL+JSON](https://en.wikipedia.org/wiki/Hypertext_Application_Language) for the REST API, the application extends the Spring HATEOAS module's [RepresentationModel](https://docs.spring.io/spring-hateoas/docs/current/reference/html/#fundamentals.representation-models) class with `AccountModel`. Like the `Account` class, its attributes represent a row of data in the `accounts` table.
 
-The contents of [`AccountModel.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/AccountModel.java):
+The contents of [`AccountModel.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/jpa/AccountModel.java):
```
Comment (ericharmeling) [inline v20.1/build-a-spring-app-with-cockroachdb-jpa.md:645]: The url is now actually

https://github.com/cockroachlabs/roach-data/tree/master/roach-data-jpa/src/main/java/io/roach/data/json/AccountModel.java

File: v20.1/build-a-spring-app-with-cockroachdb-jpa.md [hunk 3]
```diff
@@ -655,7 +655,7 @@ We don't go into much detail about hypermedia representation in this tutorial. F
 
 To abstract the database layer, Spring applications use the [`Repository` interface](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories), or some subinterface of `Repository`. This interface maps to a database object, like a table, and its methods map to queries against that object, like a [`SELECT`](selection-queries.html) or an [`INSERT`](insert.html) statement against a table.
 
-[`AccountRepository.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/AccountRepository.java) defines the main repository for the `accounts` table:
+[`AccountRepository.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/jpa/AccountRepository.java) defines the main repository for the `accounts` table:
```
Comment (ericharmeling) [inline v20.1/build-a-spring-app-with-cockroachdb-jpa.md:658]: The url is now actually

https://github.com/cockroachlabs/roach-data/tree/master/roach-data-jpa/src/main/java/io/roach/data/json/AccountRepository.java

File: v20.1/build-a-spring-app-with-cockroachdb-jpa.md [hunk 4]
```diff
@@ -672,7 +672,7 @@ For details about control flow and transaction management in this application, s
 
 ### REST controller
 
-There are several endpoints exposed by the application's web layer, some of which monitor the health of the application, and some that map to queries executed against the connected database. All of the endpoints served by the application are handled by the `AccountController` class, which is defined in [`AccountController.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/AccountController.java):
+There are several endpoints exposed by the application's web layer, some of which monitor the health of the application, and some that map to queries executed against the connected database. All of the endpoints served by the application are handled by the `AccountController` class, which is defined in [`AccountController.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/jpa/AccountController.java):
```
Comment (ericharmeling) [inline v20.1/build-a-spring-app-with-cockroachdb-jpa.md:675]: The url is now actually

https://github.com/cockroachlabs/roach-data/tree/master/roach-data-jpa/src/main/java/io/roach/data/json/AccountController.java

File: v20.1/build-a-spring-app-with-cockroachdb-jpa.md [hunk 5]
```diff
@@ -703,7 +703,7 @@ For more details about advice ordering, see [Advice Ordering](https://docs.sprin
...
 
-In this application, transaction retry logic is written into the methods of the `RetryableTransactionAspect` class, declared an aspect with the `@Aspect` annotation. Here are the contents of [`RetryableTransactionAspect.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/RetryableTransactionAspect.java):
+In this application, transaction retry logic is written into the methods of the `RetryableTransactionAspect` class, declared an aspect with the `@Aspect` annotation. Here are the contents of [`RetryableTransactionAspect.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/jpa/RetryableTransactionAspect.java):
```
Comment (ericharmeling) [inline v20.1/build-a-spring-app-with-cockroachdb-jpa.md:706]: The url is now actually

https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/json/RetryableTransactionAspect.java

**Review by ericharmeling — CHANGES_REQUESTED** [review]
@saintmalik Thanks for fixing these links!

The URLs for JDBC look spot-on, but the JPA ones are slightly off. See my inline comments.

An additional request: could you propagate these changes to:

- v20.2/build-a-spring-app-with-cockroachdb-jdbc.md
- v20.2/build-a-spring-app-with-cockroachdb-jpa.md

Thanks

**Comment by saintmalik** [comment]
Made the fix both in v20.1 and v20.2

**Review by ericharmeling — APPROVED** [review]
@saintmalik Thanks for updating this! Looks good to me now. :)

**Comment by SWDevAngel** [comment]
@saintmalik Thank you for contributing to CockroachDB this year. As a token of our appreciation we would like to send you a gift. Please DM me on our [community Slack](https://cockroa.ch/slack) `@lisa` so I can send you a link. (If you don’t want a gift, we also have a charitable contribution choice.) All orders need to be in by December 13, so please send this asap. :)

**Pull request merged**